packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-side-note/signup-side-note-provider.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { PaymentAgentDetailsResponse } from '@deriv/api-types';
+import { Money, StaticUrl } from '@deriv/components';
+import { routes, website_name } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import { History } from 'history';
+import { TNote } from './signup-side-note';
+
+type TRestPaymentAgentdetailsFields = {
+    eligibility_validation?: string[];
+    can_apply?: 0 | 1;
+};
+
+type TGetNoteProps = {
+    country_code: string;
+    history: History;
+    initial_deposit_per_country: { [key: string]: number };
+    paymentagent_details: PaymentAgentDetailsResponse['paymentagent_details'] & TRestPaymentAgentdetailsFields;
+    openWizard: VoidFunction;
+};
+
+const getNote = ({
+    country_code,
+    history,
+    initial_deposit_per_country,
+    openWizard,
+    paymentagent_details,
+}: TGetNoteProps): TNote => {
+    const can_apply_to_be_a_payment_agent = !!paymentagent_details?.can_apply;
+    const is_applied = paymentagent_details.status === 'applied';
+    const is_authorized = paymentagent_details.status === 'authorized';
+    const is_rejected = paymentagent_details.status === 'rejected';
+    const is_verified = paymentagent_details.status === 'verified';
+    const insufficient_deposit = paymentagent_details.eligibility_validation?.some(
```

Comment:
How about creating a function for this part, since it's repeated 4 times?
`paymentagent_details.eligibility_validation?.some(
        status => status === VARIABLE_NAME
    );`